soh/include/macros.h

```diff
@@ -293,9 +293,13 @@ extern GraphicsContext* __gfxCtx;
 // #endregion
 
 // #region SOH [Enhancements]
-#define CHECK_EQUIPMENT_AGE(i, j) (CVarGetInteger("gTimelessEquipment", 0) || (gEquipAgeReqs[i][j] == 9) || (gEquipAgeReqs[i][j] == ((void)0, gSaveContext.linkAge)))
-#define CHECK_SLOT_AGE(slotIndex) (CVarGetInteger("gTimelessEquipment", 0) || (gSlotAgeReqs[slotIndex] == 9) || gSlotAgeReqs[slotIndex] == ((void)0, gSaveContext.linkAge))
-#define CHECK_ITEM_AGE(itemIndex) (CVarGetInteger("gTimelessEquipment", 0) || (gItemAgeReqs[itemIndex] == 9) || (gItemAgeReqs[itemIndex] == gSaveContext.linkAge))
+#define AGE_REQ_ADULT LINK_AGE_ADULT
+#define AGE_REQ_CHILD LINK_AGE_CHILD
+#define AGE_REQ_NONE 9
+
+#define CHECK_EQUIPMENT_AGE(i, j) (CVarGetInteger("gTimelessEquipment", 0) || (gEquipAgeReqs[i][j] == AGE_REQ_NONE) || (gEquipAgeReqs[i][j] == ((void)0, gSaveContext.linkAge)))
+#define CHECK_SLOT_AGE(slotIndex) (CVarGetInteger("gTimelessEquipment", 0) || (gSlotAgeReqs[slotIndex] == AGE_REQ_NONE) || gSlotAgeReqs[slotIndex] == ((void)0, gSaveContext.linkAge))
+#define CHECK_ITEM_AGE(itemIndex) (CVarGetInteger("gTimelessEquipment", 0) || (gItemAgeReqs[itemIndex] == AGE_REQ_NONE) || (gItemAgeReqs[itemIndex] == gSaveContext.linkAge))
```

Comment:
Worth noting that these macros were recently added on decomp side, and they elected to put them in `z_kaleido_scope.h` as they are only ever used in the kaleido code. We are also not using them anywhere else, so we could probably do the same
https://github.com/zeldaret/oot/blob/main/src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.h

Reply:
yea i just didnt want to figure out the includes tbh, i think we end up using some of the macros in z_player and the age req macros are in z_play. to begin with i wasnt sure how much i was gonna need to add them but can just include the kaleido header for those files 👍🏼

Reply:
I did a search all for these macros and did not see them used outside of kaleido files.

Reply:
i meant i added them in this PR, theyre only used i think in cases where they were added in SoH

Reply:
oh.. it would help if I searched in _this_ branch, not develop lol

Reply:
haha, done now 👍🏼